docs/howto/getting-started/change-cust-billing.md

```diff
@@ -0,0 +1,31 @@
+# How to change company information in your account
+
+Via {{gui}} <https://{{gui_domain}}> you can change contact person, adress, company name, purchase order or add c/o adress. This guide will show you how.
+
+## Prerequisites
+
+You need a Cleura account.
+
+[Guide how to create a new account](https://docs.cleura.cloud/howto/getting-started/create-account)
+
+## How to change in Cleura Controlpanel
+
+To get started, navigate to <https://{{gui_domain}}>.
+Log in and click on the _Profile_ button at the top right like picture below.
+
+![picture-profile](assets/change-cust-billing/picture-profile.png)
```

Comment:
"picture-profile" is not useful as [alt text](https://accessibility.huit.harvard.edu/describe-content-images); replace this with a proper image description.